src/compiler/watch.ts

```diff
@@ -642,7 +642,7 @@ namespace ts {
         }
 
         function computeHash(data: string) {
-            return system.createHash ? system.createHash(data) : data;
+            return system.createHash ? system.createHash(data) : generateDjb2Hash(data);

```

Comment:
just use `createHash` when available. you already made it use `generateDjb2Hash` in sys. watch is no triggered except on node anyways.